week-02-ajax-and-js-objects/day-02/drill/readme.md

```diff
@@ -0,0 +1,48 @@
+## HTML Strings
+
+For the following challanges you are going to use the template from https://github.com/sf-wdi-24/frontend-template
+
+1. From the template repo copy the url to clone it
+2. In your terminal, go into your dev folder and type 'git clone' + paste the link
+3. Open this up in Sublime
+4. In `main.js` change the file from:
+    ```
+    // wait for DOM to load before running JS
+    $(function() {
+
+    // check to make sure JS is loaded
+    console.log('JS is loaded!');
+
+    // your code here
+
+    });
+   ```
+   to:
+   ```
+   console.log("Sanity Check: JS is working!");
```

Comment:
Do they need to change this? It does exactly the same thing so this might be a confusing step.